format/pyrdp/pdu/client_data.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022-2023 GoSecure Inc.
+package pyrdp
+
+import (
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+const (
+	RDP4     = 0x80001
+	RDP5     = 0x80004
+	RDP10    = 0x80005
+	RDP10_1  = 0x80006
+	RDP10_2  = 0x80007
+	RDP10_3  = 0x80008
+	RDP10_4  = 0x80009
+	RDP10_5  = 0x8000A
+	RDP10_6  = 0x8000B
+	RDP10_7  = 0x8000C
+	RDP10_8  = 0x8000d
+	RDP10_9  = 0x8000e
+	RDP10_10 = 0x8000f
+)
+
+var RDPVersionMap = scalar.UintMap{
```

Comment:
If the description don't add much i usually skip it and use `scalar.UintMapSymStr`. There was some similar cases below it seems

Reply:
Fixed with 0c69df62